### ISSUE TITLE
Images uploaded via Cloudinary no longer visible

### ISSUE THREAD

**Issue by Juggern4ut** [strapi/strapi#13301]
### Required System information

- Node.js version: v16.15.0
- Strapi version: v4.1.10
- Database: PostgreSQL on Heroku
- Operating system: Heroku Linux dyno

### Describe the bug

When uploading assets using the Cloudinary provider, the upload works as intended and the images can be used correctly but I don't seem to be able to see the images in the admin-panel. Instead it just shows a gray/white grid indicating that the image is not available. 

Looking into the console I see the following error:

> Content Security Policy: The page's settings blocked the loading of a resource at inline ("script-src").

I updated the file `./config/env/production/middleware.js` to allow the image source of cloudinary, but I can't seem to get rid of that error. Also it worked fine a few days ago and I since updated Strapi from `v4.1.9` to `v4.1.10`

### Screenshots

<img width="419" alt="Screenshot 2022-05-12 171449" src="https://user-images.githubusercontent.com/12249085/168109352-40738623-7efa-4ad7-928a-ae9d8076cbf8.png">

#### ./config/env/production/middleware.js
```js
module.exports = [
  "strapi::errors",
  {
    name: "strapi::security",
    config: {
      contentSecurityPolicy: {
        directives: {
          "connect-src": ["'self'", "https:"],
          "img-src": [
            "'self'",
            "data:",
            "blob:",
            "dl.airtable.com",
            "res.cloudinary.com",
          ],
          "media-src": [
            "'self'",
            "data:",
            "blob:",
            "dl.airtable.com",
            "res.cloudinary.com",
          ],
          upgradeInsecureRequests: null,
        },
      },
    },
  },
  "strapi::cors",
  "strapi::poweredBy",
  "strapi::logger",
  "strapi::query",
  "strapi::body",
  "strapi::session",
  "strapi::favicon",
  "strapi::public",
];
```

#### ./config/env/production/plugins.js

```js
module.exports = ({ env }) => ({
  upload: {
    config: {
      provider: "cloudinary",
      providerOptions: {
        cloud_name: env("CLOUDINARY_NAME"),
        api_key: env("CLOUDINARY_KEY"),
        api_secret: env("CLOUDINARY_SECRET"),
      },
      actionOptions: {
        upload: {},
        delete: {},
      },
    },
  },
});

```

Thank you for your Help

**Comment by derrickmehaffy** [comment]
Tested on Heroku using cloudinary and Strapi v4.1.10 and cannot reproduce. We need some more debug information.

My guess is that `NODE_ENV` isn't set at all in Heroku (eg set it to production)

Will mark as closed for now until we have more information, reply back once you have some info and we can reopen it. Thanks!

**Comment by Juggern4ut** [comment]
Hello, thank you for your quick response :)

The `NODE_ENV` seems to be set correclty since it shows when visiting the base-url of strapi. Just to be sure, I also set it manually in Heroku but it did not change the behaviour. 

<img width="368" alt="Screenshot 2022-05-13 092411" src="https://user-images.githubusercontent.com/12249085/168232865-207f9731-ad52-423e-a08f-209a3e1376e3.png">

What more Information should I provide. I don't really know what comes into play except for the files `./config/env/production/plugins.js` and `./config/env/production/middleware.js` which I already posted above.

Thank you for your help
